README.md

```diff
@@ -47,7 +47,7 @@ class MyErrorHandler
       raise SomeError.new("Something went wrong")
     rescue e
       context.response.status_code = 500
-      context.response.print MyApp::ExceptionPage.for_runtime_exception(context, e).to_s
+      context.response.print MyApp::ExceptionPage.new context, e
```

Comment:
No need to call `to_s` on this since `response.print` will do it automatically?

Reply:
Correct, with the added benefit of not having to load the entire string into memory since I'm pretty sure it'll be written directly to the response IO.